src/testRunner/unittests/tsbuildWatch/programUpdates.ts

```diff
@@ -674,7 +713,36 @@ export function someFn() { }`),
                     sys.checkTimeoutQueueLength(0);
                 },
             },
-        ]
+            {
+                caption: "Modify extendsConfigFile2",
+                change: sys => sys.writeFile("/a/b/extendsConfig2.tsconfig.json", JSON.stringify({
+                    compilerOptions: { strictNullChecks: true }
+                })),
+                timeouts: sys => { // Build project3
+                    sys.checkTimeoutQueueLengthAndRun(1);
+                    sys.checkTimeoutQueueLength(0);
+                },
+            },
+            {
+                caption: "Modify project 3",
+                change: sys => sys.writeFile("/a/b/project3.tsconfig.json", JSON.stringify({
+                    extends: ["./extendsConfig1.tsconfig.json", "./extendsConfig2.tsconfig.json"],
+                    files: ["/a/b/other2.ts"]

```

Comment:
```suggestion
                    compilerOptions: { composite: false },
                    files: ["/a/b/other2.ts"],
```

so that only edit is removal of `extendsConfig3.tsconfig.json` from the file